i18n/en.json

```diff
@@ -53,7 +53,7 @@
   "activity_log.os": "OS: {os}",
   "activity_log.sessionId": "Session ID: {id}",
   "activity_log.sessionsDescription": "Sessions are created when you log in to a new browser on a device. Sessions let you use Mattermost without having to log in again for a time period specified by the System Admin. If you want to log out sooner, use the 'Logout' button below to end a session.",
-  "add_channels_to_scheme.title": "Add Channels To **Channel Selection** List",
+  "add_channels_to_scheme.title": "Add Channels to **Channel Selection** List",
```

Comment:
This requires change to defaultMessage in corresponding components where the translation is applied.

Reply:
@saturninoabril I'll open a separate PR for default messages in all of the components, if that's okay? For this PR, I was hoping to confine the changes to only one file (and one group of the strings, namely Titles)

Reply:
I'm 3/5 that it should be part of this PR.  Separating it will be hard to back track in the future.

Reply:
Hi @saturninoabril, I edited the defaultMessage in the corresponding components for all of these titles. Please have a look?